packages/config/src/__tests__/loadConfigFromFile.test.ts

```diff
@@ -157,9 +158,20 @@ describe('loadConfigFromFile', () => {
       expect(resolvedPath).toMatch(path.join(ctx.fs.cwd(), 'prisma.config.ts'))
       expect(config).toBeUndefined()
       assertErrorTypeScriptImportFailed(error)
-      expect(error.error.message.replaceAll(resolvedPath!, '<prisma-config>.ts')).toMatchInlineSnapshot(`
+
+      const { normalisedPath, columnNumber } = (() => {
+        if (process.platform === 'win32') {
+          const actualPath = fs.realpathSync.native(resolvedPath, { encoding: 'utf-8' })
+          return { normalisedPath: actualPath, columnNumber: 4 }
+        } else {
+          return { normalisedPath: resolvedPath, columnNumber: 3 }
+        }
+      })()
+
+      // eslint-disable-next-line jest/no-interpolation-in-snapshots
+      expect(error.error.message.replaceAll(normalisedPath, '<prisma-config>.ts')).toMatchInlineSnapshot(`
         "  [31mx[0m Unexpected eof
-           ,-[[36;1;4m<prisma-config>.ts[0m:5:3]
+           ,-[[36;1;4m<prisma-config>.ts[0m:5:${columnNumber}]
```

Comment:
If we're going to do this, then we should use `expect(...).toEqual(...)` and not a snapshot.